CMakeLists.txt

```diff
@@ -57,11 +57,11 @@ ament_export_targets(${PROJECT_NAME})
 ament_export_dependencies(rclcpp rcutils std_msgs)
 
 if(BUILD_TESTING)
-  find_package(ament_lint_auto REQUIRED)
+  find_package(ament_cmake_copyright)
```

Comment:
Add the follwing to the package.xml
```xml
  <test_depend>ament_cmake_copyright</test_depend>
```

CI is failing https://build.ros2.org/job/Rpr__message_filters__ubuntu_noble_amd64/16/console